cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -115,8 +141,33 @@ def _provisioner(template, provisioning_data, delayed=None):
     return _provisioner
 
 
+def check_all_tabs(provision_request, provider):
+    view = navigate_to(provision_request, "Details")
+    widgets_names = 'request', 'purpose', 'catalog', 'environment', 'schedule'
```

Comment:
```suggestion
    widget_names = 'request', 'purpose', 'catalog', 'environment', 'schedule'
```

Reply:
Uh is this correct? Once I asked I think native English speaker and he told that probably the doubles 's' is correct.

Reply:
Usually, when coding I consider variable names as their own noun. So `widget_names` makes more intuitive sense to me here. 

Technically, if we were talking about the names _of *a* widget_ here, then something like 
```
The widget's names are: request, purpose, catalog, environment,  and schedule.
```
Would be correct. Note the `'`. If we were talking about _the names of many widgets_, then something like 
```
The widgets' names are: ...
```
is correct. But generally these types of formulations are a mouthful to say so you'd usually switch it to `the names of the widgets are: ...`.

Reply:
OK. I already changed  the identifier. Thanks for the explanation.